apis/python/setup.py

```diff
@@ -128,15 +129,11 @@ def build_libtiledbvcf():
     src_dir = p.native_lib_src_dir
     os.makedirs(build_dir, exist_ok=True)
 
-    cmake_exe = os.environ.get('CMAKE', 'cmake')
-    cmake_cmd = [cmake_exe,
-                 '-DENABLE_ARROW_EXPORT=ON',
-                 '-DFORCE_EXTERNAL_HTSLIB=ON',
-                 '-DCMAKE_BUILD_TYPE=Release',
-                 src_dir]
+    cmake_cmd = [os.environ.get('CMAKE', 'cmake')]
+    cmake_cmd.extend(get_cmake_overrides())
+    cmake_cmd.append(src_dir)
 
-    env_conf = get_cmake_overrides()
-    cmake_cmd.extend(env_conf)
+    print("CMake configure command: {}".format(cmake_cmd))
```

Comment:
Left over debug or is the print on purpose?

Reply:
Left over on purpose. I noticed we use them in [tiledb-py](https://github.com/TileDB-Inc/TileDB-Py/blob/d32b65667f8a10fce46be2e54ce5352178778385/setup.py#L205) and thought the feedback was helpful for options set via env variables.

No problem to take them out.